templates/eap-xp1-amq-s2i.json

```diff
@@ -0,0 +1,917 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,eap-xp,javaee,java,jboss,hidden",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss EAP XP + AMQ 7",
```

Comment:
Maybe "Template For JBoss EAP XP and JBoss AMQ"

Reply:
JBoss EAP XP and JBoss AMQ Template?